app/src/main/res/layout/movie_details_fragment.xml

```diff
@@ -116,7 +128,8 @@
                     android:gravity="center"
                     android:text="@string/budgetLabel"
                     android:textSize="20sp"
-                    android:textStyle="bold" />
+                    android:textStyle="bold"
+                    app:isVisible="@{viewModel.isScreenElementsVisible}"/>
```

Comment:
DRY rule seems to be broken here :) 
You can avoid such repetition by using a `Group` widget from `ContraintLayout`

Reply:
Ok I changed it :D